renderer.js

```diff
@@ -34,8 +39,9 @@ const getFileTitle = parsedMarkdown =>
   parsedMarkdown
     .chain(m => Maybe.fromNullable(m.headings))
     .map(headings => headings[0])
+    .filter(title => !!title)
     .map(title => title.trim())
-    .getOrElse('NO_NAME')
+    .getOrElse(null)
```

Comment:
Instead of `getOrElse(null)`, just remove it and change line 33 for `getFileTitle(parsedMarkdown).getOrElse(getFileName(filePath))` or split it in two lines if you want to keep it simple (3 `get`s on the same line is not decent :P)

The `Maybe` thing is a concept to remove the whole `null` and null check business.
I have to admit that it's usefulness is not amazing in a dynamically typed language but let's not mixed maybes and null :)